sdk/ai/Azure.AI.Agents.Persistent/tests/Samples/Sample_PersistentAgents_As_IChatClient.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
Please add the markdown file for the sample to the [samples](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/ai/Azure.AI.Agents.Persistent/samples) folder. We need to include all the code in `#region Snippet:<...>` into the files.
In the samples we only need to mark snippets as
\`\`\`C# Snippet:PersistentAgentsAsIChatClient_CreateClient
\`\`\`
And then run the next script to populate the snippets. All snippets in the code needs to be used in the md file. 
```bash
eng\scripts\Update-Snippets.ps1 ai\Azure.AI.Agents.Persistent
```